src/main.js

```diff
@@ -56,26 +57,36 @@ function isHidden(e) {
     style.display === "none" ||
     style.opacity === "0" ||
     style.fontSize === "0px" ||
-    style.visibility === "hidden"
+    style.visibility === "hidden" ||
+    style.position === "absolute"
   ) {
     return true;
   }
   return false;
 }
 
 function getTextFromElement(e) {
-  return e.innerText === "" ? e.dataset.content : e.innerText;
+  return (e.innerText === "" ? e.dataset.content : e.innerText) || "";
+}
+
+function getTextFromContainerElement(e) {
+  // we only need the data-content of a container element, or any direct text inside it
+  return e.dataset.content || Array.prototype.filter.call(e.childNodes, function (element) {
+      return element.nodeType === Node.TEXT_NODE;
+  }).map(function (element) {
+      return element.textContent;
+  }).join("");
```

Comment:
nit: you can rewrite this using `=>` syntax.

Reply:
sure. I'll fix that with the next push

Reply:
fixed